src/components/connectAPI.js

```diff
@@ -0,0 +1,53 @@
+export const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/Ze0tRMOWnvW25f5ddSv1/books/';
+
+export const getBooks = async () => {
```

Comment:
- Kindly use [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) with [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) or [axios](https://axios-http.com/docs/intro) for making API calls.